operator/v1/types_ingress.go

```diff
@@ -1097,7 +1097,7 @@ type LoggingDestination struct {
 // IngressControllerCaptureHTTPHeader describes an HTTP header that should be
 // captured.
 type IngressControllerCaptureHTTPHeader struct {
-	// name specifies a header name.  Its value must be a valid HTTP header
+	// name specifies a header name to be set or delete. Its value must be a valid HTTP header
```

Comment:
I'm not sure about this change, the type already existed and is being used elsewhere, I don't think the name is exclusively for being `set or deleted`, I think this line should be reverted

Reply:
ok